src/components/filters/Filters.jsx

```diff
@@ -26,6 +26,10 @@ export default class Filters extends React.Component {
     this.resetFilters = this.resetFilters.bind(this);
   }
 
+  componentWillMount() {
+    this.props.getFiltersOptions();
+  }
+
```

Comment:
Each time the component is mounted, filter options will be downloaded. I would move this logic to MapPage, and control if they have already been fetched (we are already doing this with projects, take a look at it, please);